elements/game/renderer.js

```diff
@@ -1,6 +1,6 @@
 const _ = require('lodash')
 const assets = require('../../lib/utils/assets')
-const game = require('../../lib/utils/game')
+const game = require('../../lib/game/index')
```

Comment:
Is there a reason we require the index.js file directly? Couldn't we just require /game ?